### PR TITLE
html-forms - Lamberto Martinez

### PR DESCRIPTION
**Pull request by Lambpato**
Screenshot 2023-01-24 at 9 58 15 AM

### CODE REVIEW

File: html-forms/index.html [hunk 1]
```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Forms</title>
+  </head>
+  <body>
+    <h1>Coding Form!</h1>
+    <p>Email
+      <br/>
```
Comment (shawnkost) [inline html-forms/index.html:9]: You should avoid using the <br/> element as much as possible. If you want your text to appear on a new line and not next to other elements, you should be using a <div> instead with a <p> tag inside.

File: html-forms/index.html [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+      <input type="text" name="email" />
```
Comment (shawnkost) [inline html-forms/index.html:10]: All inputs that have text content should be using a label element either by using a "for" attribute to connect the two, or by wrapping the input inside a label.

**Review by shawnkost — CHANGES_REQUESTED** [review]
Hey Lamberto! A few changes are required to finish this exercise off. You should avoid using br elements as much as possible, and all inputs that have text content, should have an associated label element tied to it.

File: html-forms/index.html [hunk 3]
```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Forms</title>
+  </head>
+  <body>
+
+    <h1>Coding Form!</h1>
+
```
Comment (shawnkost) [inline html-forms/index.html:9]: You are missing the `<form>` element. All of the `<input>` & `<textarea>` should be wrapped inside of your `<form>` element

File: html-forms/index.html [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+    <div>
+      <label for="email">Email</label>
+    </div>
+
+    <input type="text" name="email" />
```
Comment (shawnkost) [inline html-forms/index.html:14]: In order to attach an input to a label, the `<input>` element should have an id attribute that matches the value of the "for" attribute. So in this case, your input needs an `id="email"` in order to attach it to your label

File: html-forms/index.html [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+
+    <div>
+      Why is coding cool?
```
Comment (shawnkost) [inline html-forms/index.html:17]: This should also become a `<label>` just like what you did on line 11

File: html-forms/index.html [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+    </div>
+
+    <select name="Select an Option">
```
Comment (shawnkost) [inline html-forms/index.html:20]: Give this `<select>` an id to connect to the `<label>` element you will create above ^

File: html-forms/index.html [hunk 7]
```diff
@@ -0,0 +1,55 @@
...
+      <option value="Because I can code a girlfriend">Because I can code a girlfriend</option>
+    </select>
+
+    <p>
+      Favorite Language (choose one)
+    </p>
+
+    <input type="radio" name="language" value="HTML" /> HTML
+    <input type="radio" name="language" value="CSS" /> CSS
+    <input type="radio" name="language" value="JavaScript" /> JavaScript
```
Comment (shawnkost) [inline html-forms/index.html:34]: These 3 `<input>` for the various languages each need their own unique `<label>`

File: html-forms/index.html [hunk 8]
```diff
@@ -0,0 +1,55 @@
...
+
+    <p>
+      Would you like to sign up for more information?
+    </p>
+
+    <input type="checkbox" name="Sign-Up" value="Sign me up!"/>
```
Comment (shawnkost) [inline html-forms/index.html:40]: Needs a `<label>` associated to this `<input>`

File: html-forms/index.html [hunk 9]
```diff
@@ -0,0 +1,55 @@
...
+    Sign me up!
+
+    <div>
+      Comments
```
Comment (shawnkost) [inline html-forms/index.html:44]: Turn this into a `<label>` like you did on line 11

File: html-forms/index.html [hunk 10]
```diff
@@ -0,0 +1,55 @@
...
+    </div>
+
+    <textarea rows="10" cols="35" id="comments">  </textarea>
```
Comment (shawnkost) [inline html-forms/index.html:47]: Attach this to the newly created `<label>` element above ^

**Review by shawnkost — CHANGES_REQUESTED** [review]
Hey Lamberto! A few more changes requested here. Make sure you're following the instructions correctly on the exercise about giving each `<input>` a `<label>`

File: html-forms/index.html (hunk 9 above)
Comment (Lambpato) [reply html-forms/index.html:44]: I realized that soon as i sent the last one haha changes have been made

**Review by JenBowman1 — APPROVED** [review]
👍

**Pull request merged**